lib/ffmpeg/decoderbase.py

```diff
@@ -27,7 +27,6 @@ class Decoder(PropertyHandler, BaseFormatMapper):
   decoded   = property(lambda s: s._decoded)
   osdecoded = property(lambda s: filepath2os(s.decoded))
   hwaccel   = property(lambda s: s.props["hwaccel"])
-  strict    = property(lambda s: s.ifprop("strict", "-strict {strict}"))
```

Comment:
We don't need to remove this property from the decoder.  We might need to use it again in the future for something else.  If it is not set by the "test", then it just resolves to an empty string (therefore, not added to command-line).  #687 removes this from the "test" settings... that is enough to remove the flag from the command-line.